pkg/kubelet/secret/secret_manager.go

```diff
@@ -67,6 +69,8 @@ func (s *simpleSecretManager) RegisterPod(pod *v1.Pod) {
 func (s *simpleSecretManager) UnregisterPod(pod *v1.Pod) {
 }
 
+type GetObjectTTLFunc func() (time.Duration, bool)
```

Comment:
I'll review everything, but it'd be better if you find someone who's more well-versed in controllers to take a look at that part.